WT-2828 wtperf long test changes

sueloverso opened the pull request:
Make long test reflect MongoDB rather than riak

agorrod commented on bench/wtperf/runners/500m-btree-50r50u.wtperf:
```diff
@@ -1,19 +1,21 @@
-checkpoint_interval=60
+# Set cache to half of memory.  Enable logging and checkpoints.  Collect
+# wiredtiger stats for ftdc.
+conn_config="cache_size=16G,checkpoint=(wait=60,log_size=2GB),session_max=1024,log=(enabled),statistics=(fast),statistics_log=(wait=30,json)"
```
I'd prefer an eviction worker configuration in here.

It would also be good to add a table_config that sets page sizes explicitly

sueloverso commented:
@agorrod Please review.  I updated the conn_config to reflect MongoDB: eviction threads, session_max=20k.  I updated the table_config (only in the populate config since that is where it is created) to take a union of what MongoDB does for collections and indexes:
`internal_page_max=16K,type=file,leaf_page_max=16K,memory_page_max=10M,split_pct=90`

Once this is in, I plan to set up a Jenkins perf job to run weekly and create a few plots:
* Track number of > 2 second latency warnings and latency values.
* Track average min/max throughput to show if dropouts occurred.
* Track overall throughput.

These configs will benefit from WT-2830.

michaelcahill commented on bench/wtperf/runners/wtperf_run.sh:
```diff
@@ -22,11 +22,16 @@ runmax=$2
+		create=0
+		wtarg=$wtarg2
+	fi
 fi
```
Looks like this is close to needing `getopt`?

agorrod commented:
@sueloverso These changes look good to me.

michaelcahill commented:
Thanks @sueloverso for the change and @agorrod for the review.  I'll merge.
